libs/designsystem/page/src/page.component.ts

```diff
@@ -618,6 +616,12 @@ export class PageComponent
       if (this.header.titleClick.observed && this.hasInteractiveTitle === undefined) {
         this.hasInteractiveTitle = true;
       }
+
+      // Set document title immediately if present, then subscribe to changes
+      this.setHtmlDocTitle();
+      this.header.title$.pipe(takeUntil(this.ngOnDestroy$)).subscribe(() => {
```

Comment:
Won't this result in a redundant setHtmlDocTitle call, If the header has a title? Consider adding skip(1) to your pipe
```suggestion
      this.header.title$.pipe(skip(1), takeUntil(this.ngOnDestroy$)).subscribe(() => {
```